packages/@react-aria/button/package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@react-aria/focus": "^3.18.4",
     "@react-aria/interactions": "^3.22.4",
+    "@react-aria/toolbar": "3.0.0-beta.10",
```

Comment:
Not super ideal that the button package now depends on toolbar but I didn't want to make a whole new package for this...